modules/odc-importing-codebase-from-git-to-create-application.adoc

```diff
@@ -15,7 +15,24 @@ The following procedure walks you through the *From Git* option in the *Develope
 * *Context Dir* to specify the subdirectory for the application source code you want to use to build the application.
 * *Source Secret* to create a *Secret Name* with credentials for pulling your source code from a private repository.
 
-. In the *Builder* section, after the URL is validated, an appropriate builder image is detected, indicated by a star, and automatically selected. For the `https://github.com/sclorg/nodejs-ex` Git URL, the Node.js builder image is selected by default. If a builder image is not auto-detected, select a builder image. If required, you can change the version using the *Builder Image Version* drop-down list.
+. *Optional*: You can import a Devfile or a Dockerfile through your Git repository to further customize your deployment.
+Devfile::
+If your Git repository contains a Devfile it would be automatically detected.
+* The Devfile would be automatically detected and populated on the *Devfile path* field.
+* To edit the file import type, click *Edit import strategy* option, to select a different strategy.
+* If multiple Devfiles are detected, the Devfile path would need to be specified, relative to the *Context Dir* field, to import a specific Devfile.
+
+Dockerfile::
+If your Git repository contains a Dockerfile it would be automatically detected.
+* The Dockerfile would be automatically detected and populated on the *Dockerfile path* field.
+* To edit the file import type, click *Edit import strategy* option, to select a different strategy.
```

Comment:
Can this sentence be simplified by combining the two "to" clauses, like with line 22?
If "Edit import strategy" is an option among others, would "select" be a better word choice than "click"?